src/xref2/subst.ml

```diff
@@ -698,12 +706,17 @@ and include_ s i =
   {
     i with
     decl = include_decl s i.decl;
-    expansion_ = apply_sig_map s i.expansion_.items i.expansion_.removed;
+    expansion_ =
+      apply_sig_map s i.expansion_.items i.expansion_.removed
+        i.expansion_.compiled;
```

Comment:
There could be a `apply_sig_map'` function taking care of unpacking the signature fields

Reply:
done!